docs/1-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -248,3 +271,165 @@ spec:
   query: SELECT * FROM bitlocker_info WHERE protection_status = 0;
   purpose: Informational
   remediation: N/A
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get disk encryption status
+  platforms: macOS, Linux
+  description: Disk encryption status and information.
+  query: SELECT * FROM disk_encryption;
+  purpose: Informational
+  remediation: N/A
+  contributors: anelshaer
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Detect unencrypted SSH keys for local accounts
+  platforms: macOS, Linux, Windows, FreeBSD
+  description: Identify SSH keys created without a passphrase which can be used in Lateral Movement (MITRE. TA0008)
+  query: SELECT uid, username, description, path, encrypted FROM user_ssh_keys cross join users using (uid) WHERE encrypted=0;
+  purpose: Detection
+  remediation:
+    - User awareness about the impact of SSH Keys
+    - Rotate the identified Keys
+  contributors: anelshaer
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Detect unencrypted SSH keys for domain joined accounts
+  platforms: macOS, Linux, Windows, FreeBSD
+  description: Identify SSH keys created without a passphrase which can be used in Lateral Movement (MITRE. TA0008)
+  query: SELECT uid, username, description, path, encrypted FROM user_ssh_keys cross join users using (uid) WHERE encrypted=0 and username in (SELECT distinct(username) FROM last);
+  purpose: Detection
+  remediation:
+    - User awareness about the impact of SSH Keys
+    - Rotate the identified Keys
+  contributors: anelshaer
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get crontab jobs
+  platforms: macOS, Linux
+  description: Line parsed values from system and user cron/tab.
+  query: SELECT * FROM crontab;
+  purpose: Informational
+  remediation: N/A
+  contributors: anelshaer
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get suid binaries
+  platforms: macOS, Linux
+  description: suid binaries in common locations.
+  query: SELECT * FROM suid_bin;
+  purpose: Informational
+  remediation: N/A
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Detect dynamic linker hijacking (MITRE. T1574.006)
+  platforms: macOS, Linux
+  description: Detect any processes that run with an LD_PRELOAD or DYLD_INSERT_LIBRARIES environment variable
+  query: SELECT env.pid, env.key, env.value, p.name,p.path, p.cmdline, p.cwd FROM process_envs env join processes p USING (pid) WHERE key IN ('LD_PRELOAD', 'DYLD_INSERT_LIBRARIES');
```

Comment:
I get some results that seem to be related to Firefox extensions on my machine. Should we be excluding these?

```
    pid = 574
    key = DYLD_INSERT_LIBRARIES
  value = /Applications/Firefox.app/Contents/MacOS/libplugin_child_interpose.dylib
   name = plugin-container
   path = /Applications/Firefox.app/Contents/MacOS/plugin-container.app/Contents/MacOS/plugin-container
cmdline = /Applications/Firefox.app/Contents/MacOS/plugin-container.app/Contents/MacOS/plugin-container -childID 1 -isForBrowser -prefsLen 1 -prefMapSize 254602 -sbStartup -sbAppPath /Applications/Firefox.app -sbLevel 3 -sbAllowAudio -sbAllowWindowServer -parentBuildID 20210504152106 -appdir /Applications/Firefox.app/Contents/Resources/browser -profile /Users/zwass/Library/Application Support/Firefox/Profiles/xohjbjfb.default-release 397 gecko-crash-server-pipe.397 org.mozilla.machname.883196284 tab
    cwd = /
... more like that
```

Reply:
although , i meant to provide the queries for the other users, and they customize it based on their environment. but we can have that excluded. and i would propose to have a yaml key for `falsepositive` 

```yaml
---
apiVersion: v1
kind: query
spec:
  name: Detect dynamic linker hijacking (MITRE. T1574.006)
  platforms: macOS, Linux
  description: Detect any processes that run with an LD_PRELOAD or DYLD_INSERT_LIBRARIES environment variable
  query: SELECT env.pid, env.key, env.value, p.name,p.path, p.cmdline, p.cwd FROM process_envs env join processes p USING (pid) WHERE key IN ('LD_PRELOAD', 'DYLD_INSERT_LIBRARIES') AND env.value != '/Applications/Firefox.app/Contents/MacOS/libplugin_child_interpose.dylib';
  purpose: Detection
  falsepositive:
    - firefox libplugin
  remediation:
    - Identify the process/binary detected
    - Confirm with system owner
  contributors: anelshaer
```

Reply:
i have pushed that, although i dont feel well about that exclusion 😄

Reply:
I recommend we remove exclusions and keep it for the end user to exclude what they see fit their environment. What do you think?

Reply:
Sorry I missed these comments somehow. I think you removed the change anyway? Let me know if there's any follow up needed.

Reply:
Yes that one was resolved, i just had a question why users table should be on the left of the cross join?